### PR TITLE
🌍: Portuguese and Spanish Translation - Update

### PR DESCRIPTION
**Pull request by itzraiss**
## Summary

🌍: Update and fix the Spanish and Brazilian Portuguese languages for new features, as well as make the language more natural

## Change Type

- [x] Translation Update / fix

### CODE REVIEW

File: client/src/localization/languages/Es.tsx [hunk 1]
```diff
@@ -138,131 +142,154 @@ export default {
...
+com_nav_lang_italian: 'Italiano',
+com_nav_lang_polish: 'Polski',
+com_nav_lang_brazilian_portuguese: 'Português Brasileiro',
+com_nav_lang_russian: 'Русский',
+com_nav_lang_japanese: '日本語',
+com_nav_lang_swedish: 'Svenska',
+com_nav_lang_korean: '한국어',
+com_nav_lang_vietnamese: 'Tiếng Việt',
+com_nav_lang_traditionalchinese: '繁體中文',
+com_nav_lang_arabic: 'العربية',
```
Comment (berry-13) [inline client/src/localization/languages/Es.tsx]: and remove these

File: client/src/localization/languages/Br.tsx [hunk 2]
```diff
@@ -254,11 +256,27 @@ export default {
...
+  com_nav_lang_italian: 'Italiano',
+  com_nav_lang_polish: 'Polski',
   com_nav_lang_brazilian_portuguese: 'Português Brasileiro',
+  com_nav_lang_russian: 'Русский',
+  com_nav_lang_japanese: '日本語',
+  com_nav_lang_swedish: 'Svenska',
+  com_nav_lang_korean: '한국어',
+  com_nav_lang_vietnamese: 'Tiếng Việt',
+  com_nav_lang_traditionalchinese: '繁體中文',
+  com_nav_lang_arabic: 'العربية',
```
Comment (berry-13) [inline client/src/localization/languages/Br.tsx]: remove these translations

**Review by Berry-13 — CHANGES_REQUESTED** [review]

File: client/src/localization/languages/Br.tsx (hunk 2 above)
Comment (itzraiss) [reply client/src/localization/languages/Br.tsx]: Done

**Review by Berry-13 — APPROVED** [review]
Thank you!

**Review by danny-avila — APPROVED** [review]

**Comment by danny-avila** [comment]
for your information, there were duplicate keys included in this PR. Please make sure to delete the old translations you are replacing, or replace them in place.

```
[plugin:vite:esbuild] Duplicate key "com_endpoint_google_topk" in object literal
143|    com_endpoint_default: 'predeterminado',
144|    com_endpoint_top_p: 'Top P',
145|    com_endpoint_google_topk:
   |    ^
146|      'Top-k cambia cómo el modelo selecciona tokens para la salida. Un top-k de 1 significa que el token seleccionado es el más probable entre todos los tokens del vocabulario del modelo (también llamado decodificación codiciosa), mientras que un top-k de 3 significa que el siguiente token se selecciona entre los 3 tokens más probables (usando la temperatura).',
147|    com_endpoint_google_maxoutputtokens:

[plugin:vite:esbuild] Duplicate key "com_endpoint_google_maxoutputtokens" in object literal
145|    com_endpoint_google_topk:
146|      'Top-k cambia cómo el modelo selecciona tokens para la salida. Un top-k de 1 significa que el token seleccionado es el más probable entre todos los tokens del vocabulario del modelo (también llamado decodificación codiciosa), mientras que un top-k de 3 significa que el siguiente token se selecciona entre los 3 tokens más probables (usando la temperatura).',
147|    com_endpoint_google_maxoutputtokens:
   |    ^
148|      'Número máximo de tokens que se pueden generar en la respuesta. Especifica un valor menor para respuestas más cortas y un valor mayor para respuestas más largas.',
149|    com_endpoint_google_custom_name_placeholder: 'Establece un nombre personalizado para PaLM2',

[plugin:vite:esbuild] Duplicate key "com_endpoint_google_custom_name_placeholder" in object literal
147|    com_endpoint_google_maxoutputtokens:
148|      'Número máximo de tokens que se pueden generar en la respuesta. Especifica un valor menor para respuestas más cortas y un valor mayor para respuestas más largas.',
149|    com_endpoint_google_custom_name_placeholder: 'Establece un nombre personalizado para PaLM2',
   |    ^
150|    com_endpoint_prompt_prefix_placeholder: 'Establece instrucciones o contexto personalizados. Se ignora si está vacío.',
151|    com_endpoint_custom_name: 'Nombre personalizado',

[plugin:vite:esbuild] Duplicate key "com_endpoint_custom_name" in object literal
149|    com_endpoint_google_custom_name_placeholder: 'Establece un nombre personalizado para PaLM2',
150|    com_endpoint_prompt_prefix_placeholder: 'Establece instrucciones o contexto personalizados. Se ignora si está vacío.',
151|    com_endpoint_custom_name: 'Nombre personalizado',
   |    ^
152|    com_endpoint_prompt_prefix: 'Prefijo de indicación',
153|    com_endpoint_temperature: 'Temperatura',

[plugin:vite:esbuild] Duplicate key "com_endpoint_prompt_prefix" in object literal
150|    com_endpoint_prompt_prefix_placeholder: 'Establece instrucciones o contexto personalizados. Se ignora si está vacío.',
151|    com_endpoint_custom_name: 'Nombre personalizado',
152|    com_endpoint_prompt_prefix: 'Prefijo de indicación',
   |    ^
153|    com_endpoint_temperature: 'Temperatura',
154|    com_endpoint_default: 'predeterminado',

[plugin:vite:esbuild] Duplicate key "com_endpoint_temperature" in object literal
151|    com_endpoint_custom_name: 'Nombre personalizado',
152|    com_endpoint_prompt_prefix: 'Prefijo de indicación',
153|    com_endpoint_temperature: 'Temperatura',
   |    ^
154|    com_endpoint_default: 'predeterminado',
155|    com_endpoint_top_p: 'Top P',

[plugin:vite:esbuild] Duplicate key "com_endpoint_default" in object literal
152|    com_endpoint_prompt_prefix: 'Prefijo de indicación',
153|    com_endpoint_temperature: 'Temperatura',
154|    com_endpoint_default: 'predeterminado',
   |    ^
155|    com_endpoint_top_p: 'Top P',
156|    com_endpoint_top_k: 'Top K',

[plugin:vite:esbuild] Duplicate key "com_endpoint_top_p" in object literal
153|    com_endpoint_temperature: 'Temperatura',
154|    com_endpoint_default: 'predeterminado',
155|    com_endpoint_top_p: 'Top P',
   |    ^
156|    com_endpoint_top_k: 'Top K',
157|    com_endpoint_max_output_tokens: 'Tokens máximos de salida',
```

**Pull request merged**